ERCs/eip-5732/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "eip-5679",
+  "author": "Zainan Victor Zhou <zzn-ercref@zzn.im>",
+  "scripts": {
+    "start": "npx hardhat deploy_n_verify --network goerli",
```

Comment:
You could drop this network flag so it is settable at the command line (e.g. `npm run start -- --network mainnet`)

Reply:
Since we have goerli in the hardhat.config.ts I am hoping this reduces the barrier for someone to tryout the ERC ref-impl. Anyone with sufficient familarity of hardhat shall be able to hack it to deploy onto other network.